server/lorax_server/models/types.py

```diff
@@ -215,3 +216,26 @@ def from_pb(
             max_s=max_s,
             size=len(batch_inputs),
         )
+
+    @classmethod
+    def to_pb_classify(
+        self, batch, predicted_token_classes, confidence_scores, tokenizer
+    ) -> generate_pb2.ClassifyResponse:
+        results = []
+        for i, (pred, con) in enumerate(zip(predicted_token_classes, confidence_scores)):
+            res = format_ner_output(pred, con, batch.input_ids, tokenizer)
```

Comment:
I would definitely put a TODO to move this into the Rust code for the request handling thread. Reason being, the Python server is single-threaded, but this step runs on CPU, so we can easily run in the background while we proceed to the next batch for the server. But I would want to benchmark this and see where the bottleneck is first.

Reply:
sounds good